lib/dateHelpers.js

```diff
@@ -0,0 +1,11 @@
+export function formatDate(dateString) {
+  const [year, month, day] = dateString.split("-");
+  return `${day}.${month}.${year}`;
+}
+
+export const getFormattedTodaysDate = () => {
+  const today = new Date();
+  const month = `0${today.getMonth() + 1}`.slice(-2);
+  const day = `0${today.getDate()}`.slice(-2);
+  return `${today.getFullYear()}-${month}-${day}`;
+};
```

Comment:
Another 🥚

Mixing  function declarations and arrow functions isn't inherently bad, but consistency can improve readability and maintainability.


```suggestion
export function getFormattedTodaysDate () {
  const today = new Date();
  const month = `0${today.getMonth() + 1}`.slice(-2);
  const day = `0${today.getDate()}`.slice(-2);
  return `${today.getFullYear()}-${month}-${day}`;
};
```